tests/unit/api/entities/test_model_template.py

```diff
@@ -726,6 +721,56 @@ def test_entrypoints(self):
                 )
 
 
+@pytest.mark.components(OtxSdkComponent.OTX_API)
+class TestModelCategory:
+    @pytest.mark.priority_medium
+    @pytest.mark.unit
+    @pytest.mark.reqids(Requirements.REQ_1)
+    def test_model_category(self):
+        """
+        <b>Description:</b>
+        Check ModelCategory Enum class elements
+        <b>Expected results:</b>
+        Test passes if ModelCategory Enum class length, attributes and methods return expected values
+        <b>Steps</b>
+        1. Check ModelCategory length
+        2. Check ModelCategory elements value attribute
+        3. Check ModelCategory str method
+        """
+        assert len(ModelCategory) == 4
+        assert ModelCategory.SPEED.value == 1
```

Comment:
It's just a question. Is it necessary to check that each enum has which integer value?

Reply:
I don't think so, but I've tried to follow the existing convention for consistency.